### PR TITLE
Adjustments in HTML for starter template

### PR DESCRIPTION
**Pull request by pogadev18**
I think it would be a nice idea to have in the HTML right from the beginning an example / skeleton of
the card structure. This way, the mentees can visualize and understand how and what we are going to build
in a "dynamic" way using JavaScript DOM manipulation.

After we give them a short explination of the HTML card structure, we can ask them to delete the `<li>` tag and start
from a "clean plate".

Also, I moved the `<script>` tag in `<head>` with the `defer` attribute. Maybe we can explain to our mentees how and why we can do it this way and also showing them the possibility to move it before the closing `</body>` tag.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,16 +6,30 @@
...
     <header>
       <h1>Welcome to the JSHeroes Bootcamp!</h1>
     </header>
     <main>
-      <ul class="repo-list">
-        <!-- repo list -->
-      </ul>
+      <header>
+        <h1>GitHub Explorer</h1>
+      </header>
```
Comment (alexnm) [inline src/index.html]: I would replace the existing header/h1 from outside of `<main>` with this, to leave room inside the main tag for the search form and the list element. We can also use the section as you suggested here, but maybe we should introduce it only when the main tag needs to be split in two sections (so when introducing the form)

File: src/index.html [hunk 2]
```diff
@@ -6,16 +6,30 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>JSHeroes Bootcamp</title>
     <link rel="stylesheet" href="style.css" />
+    <script src='script.js' defer></script>
```
Comment (alexnm) [inline src/index.html:9]: 👍 Looks good, can open some discussions around how the browser renders HTML. Not to overload the mentees with details, but they can come back later to these examples and understand this things better.

File: src/index.html (hunk 1 above)
Comment (pogadev18) [reply src/index.html]: Yeah, it doesn't make sense to have it twice. :) Good catch

File: src/index.html [hunk 3]
```diff
@@ -6,16 +6,24 @@
...
-      <h1>Welcome to the JSHeroes Bootcamp!</h1>
+      <h1>GitHub Explorer</h1>
     </header>
     <main>
-      <ul class="repo-list">
-        <!-- repo list -->
+      <ul class="repo-cards">
+        <!--       Card example     -->
+        <li>
+          <span>test header</span>
```
Comment (andapop) [inline src/index.html:19]: maybe we wanna make this a `<h3>` ?

Comment (pogadev18) [reply src/index.html:19]: Good idea! Fixed it :)

Comment (alexnm) [reply src/index.html:19]: Not sure this is a good idea as headings should be in the correct order, so the semantically correct tag could be h2. However, when using lists, headings are not that relevant, as they don't separate bigger sections of text/content.  I would propose keeping it a <span> with a different class for styling maybe

**Pull request merged**